source/sass/main.scss

```diff
@@ -428,6 +428,7 @@ button {
 
     &:hover {
       border: 2px solid darken($nsf-yellow, 10%);
```

Comment:
I was about to do this, then hit a snag (it's a bit more complex). In some cases, it's actually turning a single bottom border, to an all around border on hover, so some are not actually just updating the colour.

We should open a new pull for this, I suspect.